src/QsCompiler/Compiler/CompilationLoader.cs

```diff
@@ -171,6 +179,12 @@ public struct Configuration
             internal bool SerializeSyntaxTree =>
                 BuildOutputFolder != null || DllOutputPath != null;
 
+            /// <summary>
+            /// Indicates whether the compiler will remove if-statements and replace them with calls to appropriate intrinsic operations.
+            /// </summary>
+            internal bool ConvertClassicalControl =>

```

Comment:
>ConvertClassicalControl [](start = 26, length = 23)

Would it be better to call this IntrinsicClassicalControl? ust a little more precise. "Convert" does not give outsider an idea what is converted to what.

Reply:
This is an internal property and hence will not be visible to an outsider. Additionally, there is a documenting comment that is far more informative than the name itself. :)

Reply:
But would not it be better if even internal variables were named in a more informative way? Outsiders need to debug through this code once in a while. Would not it be better if they understood what is going on without looking through files to find the informative comment? Or let me ask the same question from the other end: is there a good reason to call it Convert rather than Intrinsic?

---
In reply to: [406999603](https://github.com/microsoft/qsharp-compiler/pull/399#discussion_r406999603) [](ancestors = 406999603)

Reply:
@avasch01 Sorry, I somehow did not see this reply when I closed the thread and merged.